pkgs/tools/backup/luckybackup/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl
+, pkgconfig, libtool
+, rsync, ssh, qmake
+}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  pname = "luckybackup";
+  version = "0.5.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/luckybackup/${version}/source/${pname}-${version}.tar.gz";
+    sha256 = "0nwjsk1j33pm8882jbj8h6nxn6n5ab9dxqpqkay65pfbhcjay0g8";
+  };
+
+  buildInputs = [ pkgconfig libtool rsync ssh ];
+
+  nativeBuildInputs = [ qmake ];
+  
+  prePatch = ''
+    for File in luckybackup.pro menu/luckybackup-pkexec \
+                menu/luckybackup-su.desktop menu/luckybackup.desktop \
```

Comment:
Please be consistent with spacing. I think 4 spaces from the prior line would be appropriate here.